### PR TITLE
feat: Added Silver Greaves, Volcanic Greaves, Purple Cloak, and Emberfruit

### PR DESCRIPTION
**Pull request by WildSlowpoke**

### CODE REVIEW

File: src/data/items.mts [hunk 1]
```diff
@@ -392,4 +397,48 @@ export const items: Record<ItemName, Item> = {
...
+    label: "Purple Cloak",
+    stats: {
+      health: 45,
+      speed: 10,
+    },
+    craft: {
+      purple_scarab: 4,
+    },
+  },
+purple_scarab: {
```
Comment (NaughtyShiba) [inline src/data/items.mts:440]: Align this text with others, and should be fine togo.
image

File: src/data/items.mts [hunk 2]
```diff
@@ -392,4 +397,48 @@ export const items: Record<ItemName, Item> = {
...
+    label: "Emberfruit",
+  },
+  purple_cloak: {
+    type: "combat_equipment",
+    label: "Purple Cloak",
+    stats: {
+      health: 45,
+      speed: 10,
+    },
+    craft: {
```
Comment (NaughtyShiba) [inline src/data/items.mts:436]: Not really craft'able, but fine for now. Will think how best to include these, so items which are acquired by buying can be also included in `total craft requirements`

**Review by NaughtyShiba — CHANGES_REQUESTED** [review]
Fix codestyle/aligment, and will be good to go. Thanks!

**Comment by WildSlowpoke** [comment]
I hope i fixed that the right way

**Pull request merged**